lib/oauth2client.js

```diff
@@ -433,7 +445,7 @@ OAuth2Client.prototype._createPostOption = function (postUrl, urlEncodedRequestF
     var postOptions = util.createRequestOptions(
         this,
     {
-            'url' : postUrl,
+            'url' : url.format(postUrl),
```

Comment:
Before fixing this I'd like to see if we can determine why it was not being sent.  I am almost positive that at one point it was definitely being sent.  If it is not being sent then we should fix it so that it is sent.